### ISSUE TITLE
Complete data set dump

### ISSUE THREAD

**Issue by autermann** [enviroCar/enviroCar-server#82]
_I want to download a complete data set dump as Linked Data._

**Comment by crstn** [comment]
Also, we would need to discuss what goes into that dump. Do we also want any user information in there? I assume it would contain any information that is accessible from the API without authentication. 

@nuest incremental changes would be very neat, but I don't really see the need for that yet – the dataset is not huge so far. I guess we could just have a simple script that triggers the dump when requested through the API, and then caches it. If anyone asks for a dump within the next 24 hours, we can use the cached one.

**Comment by nuest** [comment]
No, there should be no user information at all, we cannot provide more information than is currently available through the public API. Since we are working with dumps and Fuseki, I would also assume that we cannot use the authentication mechanism yet to allow access to specific user data.

@crstn Workflow for the dump sounds good, if somebody wants near-real-time data they can use the SPARQL-endpoint, right?

**Comment by crstn** [comment]
Well, that depends on how often we run the ETL process Deepak is implementing. If we set it up to be ran every time a new track is uploaded, then the answer is yes.

**Comment by nuest** [comment]
Are you aware of the internal eventing components that we have deployed? I.e. the processes that run on every new submitted track? @matthesrieke can help you integrate with that if it would help?

**Comment by crstn** [comment]
I guess that should be the next step once we have the full conversion implemented. Once we have that, it should be easy to trigger the conversion only on the new data.
